<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;QT to STM32 Interface&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>741</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>setup</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_send">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_port_Status">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>350</y>
        <width>451</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_send">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>port_Status</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_recive">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>431</width>
        <height>191</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>receive</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>85</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_scan_again">
         <property name="text">
          <string>scan_again</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_open">
         <property name="text">
          <string>open</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_close">
         <property name="text">
          <string>close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>21</y>
        <width>146</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_port"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_speed"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>databit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_databit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>parity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_parity"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_Tmp">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>391</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_Wet">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>390</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_send_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>120</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_dht11">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1500ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2000ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2500ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3000ms</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>380</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>주기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_send_dht11">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>400</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>DHT11 Control</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>STM32 Control</string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_led2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_send_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>led2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Exit</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
